bootloader/bootloader.go

```diff
@@ -220,18 +220,18 @@ type TrustedAssetsBootloader interface {
 	DefaultCommandLine(candidate bool) (string, error)
 
 	// TrustedAssets returns the list of relative paths to assets inside the
-	// bootloader's rootdir that are measured in the boot process in the
-	// order of loading during the boot. Does not require rootdir to be set.
-	TrustedAssets() ([]string, error)
+	// bootloader's rootdir that are measured in the boot process.
+	// Does not require rootdir to be set.
+	TrustedAssets() (map[string]bool, error)
```

Comment:
this could be changed to be a map:

relativePath => assetName

for example

Reply:
I wonder if we could use that to disambiguate conflicting base names.